### ISSUE TITLE
Issue with videos playing in Chrome

### ISSUE THREAD

**Issue by clareosp** [kaltura/mwEmbed#494]
I have a video embedded on my website using this code:

           

<script type='text/javascript' src='kaltura-html5player-widget/jquery-1.4.2.min.js'></script>

<script type='text/javascript' src='kaltura-html5player-widget/mwEmbed-player-static.js'></script>

<link type='text/css' href='kaltura-html5player-widget/mwEmbed-player-static.css' rel='stylesheet'/>

<style type="text/css">
@import url("http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.1/themes/ui-darkness/jquery-ui.css");
</style>

</script>            

<script type="text/javascript" src="http://html5.kaltura.org/js" ></script>

<table style="background-color:#FFFFFF; width:1087px; height:100%;"><tr><td bgcolor="#FFFFFF" style="vertical-align:middle; text-align:center; padding:0;">
<video id="videoId" width="1087px" height="100%" preload="auto" poster="images/video_thumb_large.gif" preload="true" autoplay loop>
        <source type="video/mp4" src="http://www.onscreenproductions.com/video/i1116_OSP_PROMO_VIDEO_020513_Crancs_3.mp4" />
        <source type="video/h264" src="http://www.onscreenproductions.com/video/i1116_OSP_PROMO_VIDEO_020513_Crancs.m4v" />
        <source type="video/webm" src="http://www.onscreenproductions.com/video/i1116_OSP_PROMO_VIDEO_020513_Crancs_5.webm" />
        <source type="video/x-ms-wmv" src="http://www.onscreenproductions.com/video/i1116_OSP_PROMO_VIDEO_020513_Crancs_4.wmv" />
        <source type="video/ogg" src="http://www.onscreenproductions.com/video/i1116_OSP_PROMO_VIDEO_020513_Crancs_2.ogv" />

        </video></td></tr></table>
              

```
  <!--[if lte IE 6]>
  <embed src="http://www.onscreenproductions.com/video/i1116_OSP_PROMO_VIDEO_020513_Crancs.wmv" width="1087px" height=""></embed>

  <![endif]-->   
```

The video plays fine in IE and Firefox however it plays about 30 seconds in chrome then freezes:

http://www.onscreenproductions.com/video.html

As far as I can see Chrome is trying to play the .ogv file. If I delete that version it plays the .webm perfectly but then firefox doesn't play anything.

Any help would be much appreciated.

Thanks

**Comment by OrenMe** [comment]
Hi @clareosp - closing due to inactivity, feel free to reopen if you need additional help.
